apps/anomaly-detection/README.md

```diff
@@ -1,11 +1,106 @@
-# Anomaly Detection
+# Anomaly Detection using Deep Learning
+
+This application demonstrates a full lifecycle of anomaly detection using supervised learning developed using deep learning and leveraging [Intel BigDL](https://software.intel.com/en-us/articles/bigdl-distributed-deep-learning-on-apache-spark) as the implementation framework. 
+
+The application consists of the following modules:
+
+* **Data publisher:** Generates simulated CPU signal data based on a probability distribution and publishes to Kafka
+* **Data collector:** Collects data from Kafka and stores them in InfluxDB for use by machine learning and visualization
+* **Data ingester:** Ingests data from InfluxDB and prepares them in CSV format for model training
+* **Model training:** Trains a deep learning model 
+* **Model publisher:** Publishes the trained model and some useful statictics to Kafka to be used by the model server
+* **Model serving:** Serving models in real time and updating them as new models will become available. Its based on the [minibook](https://www.lightbend.com/blog/serving-machine-learning-models-free-oreilly-ebook-from-lightbend) 
+* **Speculative model serving:** Serving models, leveraging speculative execution and voting decision making, in real time and updating them as new models will become available. Based on this [blog post](https://developer.lightbend.com/blog/2018-05-24-speculative-model-serving/index.html)
+
+
+# Installation
+
+## Installing Data Publisher
+
+
+## Installing Data Collector
+
+
+## Installing all Training Modules
+
+The three training modules that need to be installed are:
+
+* Data ingester (pre-training)
+* Model training
+* Model publisher (post-training)
+
+All of the above modules are installed on a DC/OS pod as 3 images. The JSON that has to be used for deployment is located in `.../fdp-sample-applications/apps/anomaly-detection/bin`. It can be deployed using the command `dcos marathon pod add training-pod.json`.
+
```

Comment:
Currently, I am installing both ingester and model serving on k8. There is a chart https://github.com/lightbend/fdp-IntelBigDLSupport/tree/develop/adchart to install it.

Reply:
👍 .. will u then update the doc accordingly ?